### ISSUE TITLE
Update settings to match designs

### ISSUE THREAD

**Issue by holmesworcester** [TryQuiet/quiet#2537]
The settings screen on Quiet desktop is still using a design from before we converged the desktop and mobile settings designs. 

We should update it to use the latest designs, where we use the mobile settings layout on desktop in a right sidebar. 

https://www.figma.com/design/0j7Nna9zWmfOSNmRmQK1Uh/Quiet-Design-Library?node-id=4391-18424&t=q2TYIk6ADV4bS2hI-4

**Comment by kingalg** [comment]
I'm not sure if this is left like this on purpose or if we forgot about changing the design for channel search, but in my opinion, this blue select stands out in both day and night mode. If I'm not mistaken we only use this blue for links and this selection. @holmesworcester is it suppose to be like this?
![image](https://github.com/TryQuiet/quiet/assets/68909168/7c2dc471-032c-433e-b516-66af6e01429f)
![Screenshot 2024-07-11 at 15 18 17](https://github.com/TryQuiet/quiet/assets/68909168/fc3eb98c-8a4e-40dd-883e-7b73f280bb3e)

**Comment by jgaylor** [comment]
Good catch. This is legacy/deprecated. We should be using the darker shade of blue. https://www.figma.com/design/0j7Nna9zWmfOSNmRmQK1Uh/Quiet-Design-Library?node-id=3797-17362&t=4gze43o33Ao1Z38G-1
